common/pb/admin_commands.proto

```diff
@@ -26,3 +27,13 @@ message Command_ReloadConfig {
 		optional Command_ReloadConfig ext = 1002;
 	}
 }
+
+message Command_AdjustMod {
+	extend AdminCommand {
+		optional Command_AdjustMod ext = 1003;
+	}
+	required string admin_name = 1;
```

Comment:
The server should get this information from the admins connection. If you use this field I could make a malicious client that would say that someone other than me performed the promotion